### PR TITLE
Solve the future size issues

### PR DESCRIPTION
**Pull request by theogf**
The issue of the size was due to SVG plots in tutorial-a containing 10K points! The notebook alone was 40MB. This PR solves this problem by returning them as PNG. A hack with DisplayAs.jl is needed to avoid the production of the SVG (even with `default(;fmt=:png)`.
Adresses partially #73

### CODE REVIEW

**Comment by theogf** [comment]
Not a concrete benchmark but it looks like documentation needs 10 min less to run too (14 vs 25)

**Comment by theogf** [comment]
This need to be merged ASAP to avoid more heavy additions to gh-pages

**Review by willtebbutt — APPROVED** [review]

File: examples/a-regression/script.jl [hunk 1]
```diff
@@ -154,7 +159,7 @@ scatter(
     markeralpha=0.1,
     label="Training Data",
 )
-plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
+DisplayAs.PNG(plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4))
```
Comment (devmotion) [inline examples/a-regression/script.jl:162]: Was just wondering if you tried something like in https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@example-block, e.g.
```suggestion
plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
DisplayAs.PNG(ans) # hide
```
or
```suggestion
plot!(-1:0.001:1, init_post; label="Initial Posterior", color=4)
savefig("posterior.png"); nothing # hide
```

Comment (theogf) [reply examples/a-regression/script.jl:162]: Oh the first one looks good! Thanks! I don't know about the second one

**Comment by st--** [comment]
matplotlib supports rasterizing just parts of a plot (specifically, dense scatter plots), whilst keeping others (such as axes, labels etc) in vector form... do you know if plots.jl or makie.jl can do that as well ?

**Comment by devmotion** [comment]
I don't think they can, at least I'm not aware of such functionality.

**Review by st-- — APPROVED** [review]
It would be nicer if only the scatterplot portion of the figure could be rasterized (leaving the axes and labels etc in vector form), but if that's not supported by the backend at the moment I'm willing to accept it as is to reduce the repo size! I do prefer devmotion's two-liners though - and maybe add a comment explaining the reason behind it?

**Pull request merged**